project/Dependencies.scala

```diff
@@ -42,7 +42,6 @@ object Dependencies {
   val ficus: ModuleID =          "com.iheart"                 %% "ficus"           % "1.4.0"
 
   val akkaActor: ModuleID =         "com.typesafe.akka"   %%  "akka-actor"           % akkaV
-  val akkaContrib: ModuleID =       "com.typesafe.akka"   %%  "akka-contrib"         % akkaV
```

Comment:
removed akkaContrib:
the version of akka needed for the latest version of OpenCensus is 2.6.1. Any other version that I put will give an error saying that the versions of akka must be the same. This includes `akkaContrib`, for which the latest version I could find is 2.5.31. Removing it did not affect unit tests and seemingly doesn't affect integration tests either.